### PR TITLE
docs(usage): remove usage of deprecated classes

### PR DESCRIPTION
**Pull request by pierreboissinot**
sees https://github.com/sonata-project/SonataCoreBundle/pull/571

## Subject

It seems there is no alternative to InvalidParameterException. In SonataNotificationBundle, the depreciation says to use the exception from SonataCoreBundle, but it'll be removed from SonataCoreBundle. 
I read https://github.com/sonata-project/SonataCoreBundle/pull/571.
So maybe remove this part of code from the docs.

I am targeting this branch, because it concerns the depreciation about 3.x.

### CODE REVIEW

**Comment by greg0ire** [comment]
Please associate your email address with your Github account, or change the email in your commits to an address already associated with it.

File: docs/reference/usage.rst [hunk 1]
```diff
@@ -77,10 +76,6 @@ logger consumer creation::
         {
             $message = $event->getMessage();
 
-            if (!in_array($message->getValue('level'), $this->types)) {
-                throw new InvalidParameterException();
```
Comment (greg0ire) [inline docs/reference/usage.rst]: I've read somewhere that the replacement would be `\RuntimeException`, so maybe switch to that?

Comment (greg0ire) [reply docs/reference/usage.rst]: Also, providing a message would be great.

Comment (pierreboissinot) [reply docs/reference/usage.rst]: Thanks.
Sorry I've messed up rebase, here is the new PR https://github.com/sonata-project/SonataNotificationBundle/pull/376

**Comment by pierreboissinot** [comment]
This PR is closed (see https://github.com/sonata-project/SonataNotificationBundle/pull/376)

**Pull request closed without merging**